runtime/lua/vim/uri.lua

```diff
@@ -3,7 +3,7 @@
 -- https://tools.ietf.org/html/rfc2732
 -- https://tools.ietf.org/html/rfc2396
 
-
+local is_windows = jit.os == 'Windows'
```

Comment:
When not using `LuaJIT`, `jit` will be null, causing error.
This is a good idea though, but we should provide a fallback when not using `jit`.

Reply:
Is it possible that jit will be removed in the future?

Reply:
What about use this `is_windows = vim.loop.os_uname().version:match("Windows")`

Reply:
I had update.Please check. Thanks.

Reply:
Yeah I quite like this version.

Reply:
Tests are failing though